src/Uno.UI/Microsoft/UI/Xaml/Controls/SplitButton/SplitButtonAutomationPeer.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.UI.Xaml.Controls;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Automation.Provider;
+
+namespace Microsoft.UI.Xaml.Automation.Peers
+{
+	public class SplitButtonAutomationPeer : FrameworkElementAutomationPeer, IExpandCollapseProvider, IInvokeProvider
+	{
+		private SplitButton _owner;
+
+		public SplitButtonAutomationPeer(SplitButton owner) : base(owner)
+		{
+			_owner = owner;			
+		}
+
+		// IAutomationPeerOverrides
+		protected override object GetPatternCore(PatternInterface patternInterface)
+		{
+			if (patternInterface == PatternInterface.ExpandCollapse ||
+				patternInterface == PatternInterface.Invoke)
+			{
+				return this;
+			}
+
+			return base.GetPatternCore(patternInterface);
+		}
+
+		protected override string GetClassNameCore() => nameof(SplitButton);
+
+		protected override AutomationControlType GetAutomationControlTypeCore() => AutomationControlType.SplitButton;
+
+		private SplitButton GetImpl() => _owner;
+
+		// IExpandCollapseProvider 
+		public ExpandCollapseState ExpandCollapseState
+		{
+			get
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a way to break out of this property accessor's recursion.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5246693)